powerauth-push-server/src/main/java/io/getlime/push/repository/PushCampaignDevicesRepository.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2016 Lime - HighTech Solutions s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getlime.push.repository;
+
+import io.getlime.push.repository.model.PushCampaignDevices;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ * Repository interface used to access push campaign database.
+ *
+ * @author Martin Tupy, martin.tupy.work@gmail.com
+ */
+
+@Repository
+public interface PushCampaignDevicesRepository extends CrudRepository<PushCampaignDevices, Long> {
```

Comment:
Could you please rename `PushCampaignDevices` to `PushCampaignDevice`? The class represents only one item in the table.